browser/about_flags.cc

```diff
@@ -37,7 +37,7 @@
 #include "third_party/blink/public/common/features.h"
 
 #if BUILDFLAG(ENABLE_BRAVE_VPN) && !BUILDFLAG(IS_ANDROID)
```

Comment:
`!BUILDFLAG(IS_ANDROID)` doesn't apply to features.h

Reply:
done